internal/qtest/test_helper.go

```diff
@@ -168,8 +184,16 @@ func PopulateDatabaseWithInitialData(t *testing.T, db database.Database, sc scm.
 	if err != nil {
 		return err
 	}
-
-	admin := CreateFakeUser(t, db, uint64(1))
+	course.OrganizationID = org.GetID()
+	// TODO(Espeland): Remember to remove myself when done testing!
+	admin := &pb.User{Login: "oleespe2"}
```

Comment:
Perhaps a better solution is to add another function, e.g., `GetGitHubLogin` in `scm/env_helper.go` to read from `GITHUB_LOGIN` environment variable. I guess such a function could be given an boolean `skip` argument if the test should be skipped if the login is required for the test itself... if it is not required for the full test, then I guess it shouldn't be skip the test, but avoid using the specific user...

Reply:
I suppose this has been implemented as `GetTestUser()`